json/keycloak.json

```diff
@@ -9,7 +9,7 @@
     "updateable": false,
     "privileged": false,
     "interface_port": "8080",
-    "documentation": null,
+    "documentation": "https://github.com/tteck/Proxmox/discussions/2922",
```

Comment:
The documentation button is now mixing official application docs and extra community-script docs (github discussions). This is fine for now, but I think eventually we should add support for having links in `notes` and move all github discussions to be direct links in `notes` instead

Reply:
100% agree, this is something that i'm looking to add after i finished the json generation page